src/style.css

```diff
@@ -0,0 +1,138 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
Create separate files for `card` and `stars` BEM blocks
![image](https://user-images.githubusercontent.com/96409563/224285634-1f62b51f-ea6b-4d77-9926-0db9f46db68e.png)